src/router/db.ts

```diff
@@ -17,16 +17,17 @@ db.get('/connect/:dbAlias', async (req, res) => {
       'dbAlias'
     ].filter(k => !req.params.hasOwnProperty(k)).join(', ')}`
   );
+  let uid, email;
```

Comment:
Why the `email` here when it doesn't appear to be used outside of the `try` block?

Reply:
because it is being used in the `catch` block with the `logUserErr` fn, I think?

Reply:
> because it is being used in the `catch` block with the `logUserErr` fn, I think?

It wasn't when I first commented. Luisfer updated the PR.

Reply:
Yes, I had forgotten to push some staged changes